docker/dev-env/Dockerfile

```
@@ -13,6 +13,8 @@ ARG USER_GID="65001"
 ARG USER_GROUP="dotcms"
 ARG USER_NAME="dotcms"
 
+ARG PG_VERSION=16
+ENV PG_VERSION=16
```

If the intention is to keep these in sync may I suggest having the ENV reference the ARG...
```suggestion
ARG PG_VERSION=16
ENV PG_VERSION=${PG_VERSION}
```